Source/NSBundle.m

```diff
@@ -2144,6 +2117,48 @@ + (NSArray *) _bundleResourcePathsWithRootPath: (NSString*)rootPath
 	  addBundlePath(array, contents, primary, subPath, language);
 	}
     }
+  
+#ifdef __ANDROID__
+  // Android: check subdir and localization directly, as AAssetDir and thereby
+  // NSDirectoryEnumerator doesn't list directories
+  NSString *originalPrimary = primary;
+  if (subPath) {
+    primary = [originalPrimary stringByAppendingPathComponent: subPath];
+    contents = bundle_directory_readable(primary);
+    addBundlePath(array, contents, primary, nil, nil);
+    
+    if (localization) {
+      primary = [primary stringByAppendingPathComponent:
+        [localization stringByAppendingPathExtension:@"lproj"]];
+      contents = bundle_directory_readable(primary);
+      addBundlePath(array, contents, primary, nil, nil);
+    } else {
+      NSString *subPathPrimary = primary;
+      enumerate = [languages objectEnumerator];
+      while ((language = [enumerate nextObject])) {
+        primary = [subPathPrimary stringByAppendingPathComponent:
+          [localization stringByAppendingPathExtension:@"lproj"]];
```

Comment:
Shouldn't this be "language" instead of "localization" here?

Reply:
Good catch, thanks. Fixed.